pheps/phep-0001.md

```diff
@@ -0,0 +1,366 @@
+
+PHEP authors should incorporate community feedback by making edits and additional pushes to their branch. They must add a new `Post-History` entry with each push. It is recommended to address as much actionable feedback as practicable with each push; however, each push may contain multiple commits for ease of review. Once a PHEP PR has been marked ready for review, PHEP authors must not rewrite history after a push, i.e. no force-pushing.
+
+If applicable, the reference implementation should be developed and tested as part of the discussion process. If changes are made based on implementation experience and user feedback, they should be noted in the PHEP.
+
+PHEP authors are responsible for managing the progress of the discussion of their PHEP, for instance by updating the PR description to highlight topics of particular interest and to provide the dates of votes on the PHEP resolution.
+
+If a PHEP undergoes a significant re-write or other major changes, the author(s) and editor may discuss closing the existing PR and opening a new one, starting from the changed version. If this occurs, the `Discussions-To` link must be updated.
+
+PHEP discussions are subject to the [PyHC Code of Conduct](https://heliopython.org/docs/code_of_conduct/).
+
+## PHEP Review & Resolution
+<a name="phep-review-resolution"></a>
+Once the authors have completed a PHEP, they may request a review for style and consistency from the PHEP editors. However, content review and acceptance of the PHEP is the responsibility of the community. It should be clear from the [discussion](#discussing-a-phep) when the community is beginning to agree: concerns have been addressed and new ones are not being raised, PR reviews with "approve" status are appearing, discussion is quieting down. At this point the PHEP author should seek formal acceptance.
+
+A PHEP is accepted by consensus of attendees (virtual or in-person) at both a PyHC regular telecon and a regular PyHC meeting, with no edits to the PHEP occurring in between. Potential acceptance of a PHEP must be placed on the agenda at least a month before the meeting, including a link to the open PR with the proposed-final language of the PHEP. Persons not in attendance at the meetings may object via comment on the PR. The PHEP may be edited up until the initial vote, for instance to incorporate discussion at the meeting. However, any edits after a vote invalidate that acceptance, and two further votes (one at a meeting, one on a telecon) must be taken.
```

Comment:
Reply [above](https://github.com/heliophysicsPy/standards/pull/22#discussion_r1419336622) goes into this. It's worth kicking around.

Reply:
Can there be a specific tag people can use to express disapproval so it doesn't get lost in the conversation? Maybe as simple as a thumbs down?
I think the comment I am being tagged about isn't as important since the process is so dependent on disapprovals rather than approvals, whether on GitHub, virtual, or in-person. I do think that anyone expressing disapproval on GitHub should make themselves available for conversation before that disapproval can be used officially. Not sure on the best way forward on this, though.